src/content/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts.mdx

```diff
@@ -31,7 +31,7 @@ To see changes made to your account (for example, user management changes), you
 
 ## Query examples [#query]
 
-The [usage UI](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing#billing-usage-ui) displays your data usage and billable user count. But to get more detail, you can use these [NRQL queries](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql). For definitions of some of the attributes used in these queries, see [Attributes](#attributes).
+The [**View your usage** UI](/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings/#pricing) displays your data usage and billable user count. If you need more detail than that UI provides, you can use these [NRQL queries](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql). For definitions of some of the attributes used in these queries, see [Attributes](#attributes).
```

Comment:
maybe "If you need more detail than the UI provides" - small suggestion. And merely a matter of preference so you can ignore.